apps/api/src/services/data-provider/ghostfolio-scraper-api/ghostfolio-scraper-api.service.ts

```diff
@@ -84,33 +84,42 @@ export class GhostfolioScraperApiService implements DataProviderInterface {
   public async getQuotes(
     aSymbols: string[]
   ): Promise<{ [symbol: string]: IDataProviderResponse }> {
+    const response: { [symbol: string]: IDataProviderResponse } = {};
+
     if (aSymbols.length <= 0) {
-      return {};
+      return response;
     }
 
     try {
-      const [symbol] = aSymbols;
```

Comment:
@FedericoComoglio The `getQuotes()` function for symbols with data source `GHOSTFOLIO` was just supporting a single symbol 🙈
As soon as you have added one more activity, the calculation was broken.